app/components/municipality-select.js

```diff
@@ -0,0 +1,34 @@
+import Component from '@glimmer/component';
+import { inject as service } from '@ember/service';
+import { restartableTask } from 'ember-concurrency';
+
+export default class MunicipalitySelectComponent extends Component {
+  @service fastboot;
+  @service store;
+  municipalities;
+
+  constructor(...args) {
+    super(...args);
+
+    if (!this.fastboot.isFastBoot) {
+      this.municipalities = this.loadMunicipalitiesTask.perform();
```

Comment:
This stores the TaskInstance instead of the actual data. TaskInstances act like promises, they can resolve to a value if that's returned from the task.